python/GafferArnold/ArnoldTextureBake.py

```diff
@@ -83,72 +83,67 @@ def __init__( self, name = "__CameraSetup" ) :
 			self["__chunkExpression"] = Gaffer.Expression()
 			self["__chunkExpression"].setExpression( inspect.cleandoc(
 				"""
-				# Locate the next point in the list of files to bake where we can split the list into chunks without
-				# seperating two files that need to get combined into the same texture
-				def nextChunkBreak( i, l ):
-					while i > 0 and i < len( l ) and (
-							l[i - 1].get("udim") == l[i].get("udim") and
-							l[i - 1].get("fileName") == l[i].get("fileName") ):
-						i += 1
-					return i
-
+				import collections
 				rawInfo = parent["__udimQuery"]["out"]
 
 				defaultFileName = parent["defaultFileName"]
 				defaultResolution = parent["defaultResolution"]
 
-				listInfo = []
+				allMeshes = collections.defaultdict( lambda : [] )
 				for udim, meshes in rawInfo.items():
 					for mesh, extraAttributes in meshes.items():
 						resolution = defaultResolution
 						if "bake:resolution" in extraAttributes:
-							resolution = extraAttributes["bake:resolution"].value
+								resolution = extraAttributes["bake:resolution"].value
 
 						fileName = defaultFileName
 						if "bake:fileName" in extraAttributes:
-							fileName = extraAttributes["bake:fileName"].value
+								fileName = extraAttributes["bake:fileName"].value
```

Comment:
Indentation again? I wondered if you were trying to fix some previous tabs vs spaces problem, but as far as I can tell it was all fine before. We've just got an additional level of indentation now...